src/explorer/editors/FileEditor.ts

```diff
@@ -39,8 +39,8 @@ export class FileEditor extends BaseEditor<IAzureNode<FileTreeItem>> {
 
     async updateData(node: IAzureNode<FileTreeItem>): Promise<string> {
         const webAppClient = nodeUtils.getWebSiteClient(node);
-        const user = await util.getWebAppPublishCredential(webAppClient, node.treeItem.site);

```

Comment:
Did you remove all usages of util.getWebAppPublishCredential? If so can you delete it from util?

Reply:
The log stream team is still using the util file in LogPointsSessionWizard.ts.  I can take out util.getWebAppPublishCredential eventually, but I'd like to make a separate PR for that.